.gitlab-ci.yml

```diff
@@ -30,65 +31,134 @@ test_frontend:
     - yarn run stylelint
     - yarn run flow --quiet
 
-test_integration_and_ui:
-  stage: test_on_dev
-  image: node:10
-  only:
-    - tmp_master_for_ci_cd
-  before_script:
-    - eval $(ssh-agent -s) && sh scripts/configure_ssh.sh
-    - sh scripts/prepare_integration_tests.sh
-  script:
-    # run integration tests
-    - yarn jest tests/integration
-    # run each ui test one by one
-    - yarn runinband
+# test_integration_and_ui:
+#   stage: test_integration
+#   image: node:10
+#   only:
+#     - tmp_master_for_ci_cd
+#   before_script:
+#     - ln -fs /usr/share/zoneinfo/UTC /etc/localtime
+#     - apt-get update
+#     - apt-get install -y sudo python python-pip python-dev build-essential
+#     - pip install virtualenv
+#     - cd $CI_PROJECT_DIR
+#     - python -mvirtualenv venv
+#     - source venv/bin/activate
+#     - pip install $(egrep '(invoke|boto3|requests|jinja2|semantic-version)' requirements.in)
+#     - "echo '_extends: aws.yaml' > invoke.yaml"
+#     - touch secrets.yaml
+#     - ln -s assembl/tasks .
+#     - inv build.prepare-integration-tests
+#   script:
+#     # run integration tests
+#     - yarn jest tests/integration
+#     # run each ui test one by one
+#     - yarn runinband
+
+# test_backend:
+#   stage: test
+#   only:
+#     - invoke
+#   before_script:
+#     - ln -fs /usr/share/zoneinfo/UTC /etc/localtime
+#     - apt-get update
+#     - apt-get install -y sudo python python-pip python-dev build-essential
+#     - pip install virtualenv
+#     - cd $CI_PROJECT_DIR
+#     - python -mvirtualenv venv
+#     - source venv/bin/activate
+#     - pip install $(egrep '(invoke|semantic-version)' requirements.in)
+#     - "echo '_extends: testing_aws.yaml' > invoke.yaml"
+#     - touch secrets.yaml
+#     - ln -s assembl/tasks .
+#     - inv sudoer.install-build-dependencies
+#     - ln -s requirements-tests.frozen.txt requirements.txt
+#     - inv build.update-pip-requirements
+#     - python setup.py develop
+#     - inv sudoer.install-database
+#     - inv build.compile-messages-catalog
+#   script:
+#     # run integration tests
+#     - source venv/bin/activate
+#     - inv -f assembl/configs/testing_aws.yaml deploy.create-local-ini
+#     - inv -f assembl/configs/testing_aws.yaml build.database-create
+#     - cat testing.ini
+#     - py.test assembl/tests
 
 build_wheel:
   stage: build
   only:
     - develop
+    - invoke
   variables:
-    DEVELOP_BRANCH: develop
-    MASTER_BRANCH: tmp_master_for_ci_cd
+    LC_ALL: C.UTF-8
+    LANG: en.UTF-8
+    GIT_STRATEGY: clone
   artifacts:
     paths:
       - dist/
   before_script:
-    - sh scripts/install_build_dependencies.sh
-    - sh scripts/install_node_and_yarn.sh
-    - sh scripts/configure_assembl_remote.sh
-    - eval $(ssh-agent -s) && sh scripts/configure_ssh.sh
-    - sh scripts/clear_apt_cache.sh
```

Comment:
In the scripts folder, install_build_dependencies.sh install_node_and_yarn.sh configure_assembl_remote.sh configure_ssh.sh clear_apt_cache.sh create_assembl_wheel.sh are unused it seems. There were replaced by inv commands I guess. Please delete those scripts if we're not using them anymore. I see bump_assembl_version_and_push.sh has been deleted, but not the others scripts.